packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.component.test.tsx

```diff
@@ -83,6 +89,140 @@ describe('VitalsBiometricsForm: ', () => {
 
     expect(bmiInput).toHaveValue(19.1);
   });
+
+  describe('Vital-Biometrics Form submission', () => {
+    let saveButton: HTMLElement;
+    let muac: HTMLElement;
+    let notes: HTMLElement;
+    let temperature: HTMLElement;
+    let respirationRate: HTMLElement;
+    let oxygenSaturation: HTMLElement;
+    let pulse: HTMLElement;
+    let systolic: HTMLElement;
+    let bmiInput: HTMLElement;
+    let weightInput: HTMLElement;
+    let heightInput: HTMLElement;
+
+    beforeEach(async () => {
+      // @ts-ignore
+      jest.useFakeTimers('modern');
+      // @ts-ignore
+      jest.setSystemTime(1638682781000); // 5 Dec 2021 05:39:41 GMT
+
+      renderForm();
+
+      heightInput = screen.getByRole('spinbutton', { name: /height/i });
+      weightInput = screen.getByRole('spinbutton', { name: /weight/i });
+      bmiInput = screen.getByRole('spinbutton', { name: /bmi/i });
+      systolic = screen.getByRole('spinbutton', { name: /systolic/i });
+      pulse = screen.getByRole('spinbutton', { name: /pulse/i });
+      oxygenSaturation = screen.getByRole('spinbutton', { name: /oxygen saturation/i });
+      respirationRate = screen.getByRole('spinbutton', { name: /respiration rate/i });
+      temperature = screen.getByRole('spinbutton', { name: /temperature/i });
+      notes = screen.getByPlaceholderText(/type any additional notes here/i);
+      muac = screen.getByRole('spinbutton', { name: /muac/i });
+      saveButton = screen.getByRole('button', { name: /Save and close/i });
+    });
+    it('renders a success toast notification upon clicking the save button', async () => {
+      const promise = Promise.resolve();
+      mockSavePatientVitals.mockResolvedValueOnce({ status: 201, statusText: 'Ok' });
+
+      userEvent.type(heightInput, '180');
+      userEvent.type(weightInput, '62');
+      userEvent.type(systolic, '180');
+      userEvent.type(pulse, '180');
+      userEvent.type(oxygenSaturation, '180');
+      userEvent.type(respirationRate, '180');
+      userEvent.type(temperature, '180');
+      userEvent.type(notes, 'patient on MDR treatment');
+      userEvent.type(muac, '180');
+
+      expect(bmiInput).toHaveValue(19.1);
+      expect(systolic).toHaveValue(180);
+
+      expect(pulse).toHaveValue(180);
+      expect(oxygenSaturation).toHaveValue(180);
+      expect(respirationRate).toHaveValue(180);
+
+      expect(temperature).toHaveValue(180);
+
+      expect(notes).toHaveValue('patient on MDR treatment');
+
+      expect(muac).toHaveValue(180);
+      userEvent.click(saveButton);
```

Comment:
Running this test locally throws this warning:

<img width="587" alt="Screenshot 2022-04-04 at 21 07 40" src="https://user-images.githubusercontent.com/8509731/161605055-9b09aad7-a9d2-4843-883a-afb77d684143.png">

To remedy it, you'd need to pass an `event` object to `savePatientVitalsAndBiometrics` in the component and then call `.preventDefault()` on it. Like so:

```js
const savePatientVitalsAndBiometrics = (event: SyntheticEvent) => {
  event.preventDefault();
  // ...
}
```